.github/workflows/tests.yml

```
@@ -0,0 +1,24 @@
+name: tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        ruby-version: [2.7, 3.0, 3.1, 3.2]
```

```suggestion
        ruby-version: [2.6, 2.7, 3.0, 3.1]
```
gemspec mentions `>=2.6.9`, so it would be good to test against that as well. Also let's wait for official release of 3.2 and add them at a later point in time in the CI?

---

I believe the tests fail on 2.6, so it'd have to be 2.6.9 specifically, I'll see if it's possible to specify that version explicitly.

---

Also, ruby 3.2 was officially released I believe on Dec 25, 2023: https://devclass.com/2023/01/04/ruby-3-2-0-released-with-webassembly-compiler-production-ready-yjit-optimization-and-much-faster-regular-expressions/#:~:text=Ruby%20developers%20got%20a%20Christmas,25th%20release%20of%20Ruby%203.2.

---

here this is the better link about the 3.2 release: https://www.ruby-lang.org/en/news/2022/12/25/ruby-3-2-0-released/#:~:text=We%20are%20pleased%20to%20announce,many%20features%20and%20performance%20improvements.

---

@ritikesh tests were failing when run against ruby 2.6 or 2.6.9 because the activerecord version we were locked into required ruby 2.7, you can see that here: https://github.com/toddnestor/blueprinter/actions/runs/4048869964/jobs/6964615879

I changed the dev dependency to just require a version of active record greater than the version that used to be locked in, and now tests pass in ruby 2.6.9, also in the time since I put this PR in ruby 3.2 became supported by the ruby setup action we're using here, so it can be added back in.  You can see tests passing on all 5 listed ruby versions here: https://github.com/toddnestor/blueprinter/actions/runs/4048936811/jobs/6964763093


since ruby 2.6.9 is the min supported we show on this gem I specified that exact version, if you'd prefer ruby 2.6 though I can change it.